api/clients/eigenda_client.go

```diff
@@ -137,11 +150,13 @@ func (m EigenDAClient) putBlob(ctx context.Context, rawData []byte, resultChan c
 		return
 	}
 
+	// TODO: should we just use uint8s directly in the config?
 	customQuorumNumbers := make([]uint8, len(m.Config.CustomQuorumIDs))
 	for i, e := range m.Config.CustomQuorumIDs {
 		customQuorumNumbers[i] = uint8(e)
 	}
 	// disperse blob
+	// TODO: can we consider passing a requestID directly to requests, to get idempotency and tracing?
```

Comment:
requestID is currently computed by disperser, that uses timestamp taken at the server side